app/src/main/java/com/onrpiv/uploadmedia/Experiment/ViewResultsActivity.java

```diff
@@ -749,18 +776,58 @@ public void getSelectPosition(float x, float y) {
             // Only multipass has vorticity values (design decision to only show one vorticity field)
             float vort = (float) multiPass.getCalibratedVorticity()[pivCoords.y][pivCoords.x];
             // because we're inverting the y axis, we also invert the v values
-            updatedText = settings.formatInfoString_physical((float) imgX, (float) dispY, u, -v, vort,
-                    (float) pivCorr.getPixelToPhysicalRatio(), (float) pivCorr.getUvConversion());
+//            updatedText = settings.formatInfoString_physical((float) imgX, (float) dispY, u, -v, vort,
+//                    (float) pivCorr.getPixelToPhysicalRatio(), (float) pivCorr.getUvConversion());
+
+            String velString = String.valueOf(pivCorr.getMag()[pivCoords.y][pivCoords.x]);
+            String xString = String.valueOf((int)((float) imgX));
+            String yString = String.valueOf((int)((float) dispY));
+            String vortString = String.valueOf(vort);
+            String uString = u + " cm/s";
+            String vString = -v + " cm/s";
+            String XYConversionString = String.valueOf((float) pivCorr.getPixelToPhysicalRatio());
+            String UVConversionString = String.valueOf((float) pivCorr.getUvConversion());
+
+            resultsVelocity.setText(velString);
+            resultsX.setText(xString);
+            resultsY.setText(yString);
+            resultsVort.setText(vortString);
+            resultsU.setText(uString);
+            resultsV.setText(vString);
+            resultsXYConversion.setText(XYConversionString);
+            resultsXYConversionRow.setVisibility(View.VISIBLE);
+            resultsUVConversion.setText(UVConversionString);
+            resultsUVConversionRow.setVisibility(View.VISIBLE);
+
         } else {
             // compile the data
             float u = (float)pivCorr.getU()[pivCoords.y][pivCoords.x];
             float v = (float)pivCorr.getV()[pivCoords.y][pivCoords.x];
             // Only multipass has vorticity values (design decision to only show one vorticity field)
             float vort = (float) multiPass.getVorticityValues()[pivCoords.y][pivCoords.x];
             // because we're inverting the y axis, we also invert the v values
-            updatedText = settings.formatInfoString_pixel((float)imgX, (float)dispY, u, -v, vort);
+//            updatedText = settings.formatInfoString_pixel((float)imgX, (float)dispY, u, -v, vort);
```

Comment:
Go ahead and delete instead of commenting out. Don't want to leave dead code in the app.

Reply:
Will do. Thanks for letting me know it's alright to delete. I'll go ahead and delete the functions used as well.